quickwit/quickwit-search/src/metrics.rs

```diff
@@ -146,6 +147,12 @@ impl Default for SearchMetrics {
                 &[],
                 ["affinity"],
             ),
+            search_after_cache_size_bytes: new_gauge(
```

Comment:
that's not a search_after cache, search_after leverage partial request cache when it can (which is not as often as it could)
it should be scrol_context or something

Reply:
thanks for the clarification, I just took the name from here: https://github.com/quickwit-oss/quickwit/blob/9d12159868d7441437f211864e0cff4d44d17867/quickwit/quickwit-search/src/service.rs#L55-L61

Just to be sure that my understanding is correct: "search after" is the name of the search functionality that skip results. To use it, you have two APIs
- `_search/scroll` endpoint that is stateful because it also freezes a list of splits while going through the pages
- the `search_after` field in `_search` that is kind of a stateless equivalent of scroll

I'm going to rename some things here and there to make this clear.

Reply:
classic pagination is the rough equivalent of an SQL `LIMIT $num_res OFFSET $num_res*$page_count` (and has the exact same inefficiency!!), search after is roughly adding a `WHERE sort_key > $last_res_sort_key ORDER BY sort_key ASC`.
scroll are the equivalent to sql stateful cursors, the first implementation was based on pagination (and i believe ES scroll still is), it now rely on search after as that's more efficient. It relying on search after is an implementation detail, and search after does not in anyway need scrolls to exist